src/config.ts

```diff
@@ -5,10 +5,12 @@ const configBoolean = z
   .enum(['true', 'false'])
   .transform((arg) => JSON.parse(arg));
 
+const configStringOrBoolean = z.union([configBoolean, z.string().default("../../../../src/middleware")])
```

Comment:
I think the better name would be something like `configMiddlewareStringOrBoolean`. Since `configStringOrBoolean` won't be reused because it sets a default value for middleware path.

Reply:
Good point, i chose the name before thinking of the default value.